app/src/main/java/app/ImageStorageController.java

```diff
@@ -21,13 +21,10 @@
 import io.micronaut.http.annotation.Post;
 import io.micronaut.scheduling.TaskExecutors;
 import io.micronaut.scheduling.annotation.ExecuteOn;
-import io.micronaut.security.annotation.Secured;
-import io.micronaut.security.rules.SecurityRule;
 
 import javax.validation.Valid;
 
 @Controller("/api/image")
-@Secured(SecurityRule.IS_ANONYMOUS)
```

Comment:
Doesn't this still need to be there so that citizens can upload photos without being logged int?